### ISSUE TITLE
E0027 needs to be updated to new format

### ISSUE THREAD

**Issue by jonathandturner** [rust-lang/rust#35200]
From: src/test/compile-fail/E0027.rs

E0027 needs a span_label, taking it from:

```
error[E0027]: pattern does not mention field `name`
  --> src/test/compile-fail/E0027.rs:20:9
   |
20 |         Dog { age: x } => {} //~ ERROR E0027
   |         ^^^^^^^^^^^^^^
```

to:
```
error[E0027]: pattern does not mention field `name`
  --> src/test/compile-fail/E0027.rs:20:9
   |
20 |         Dog { age: x } => {} //~ ERROR E0027
   |         ^^^^^^^^^^^^^^ missing field `name`
```

**Comment by silenuss** [comment]
Ok, I'm finally sitting down to work on this.

**Comment by KiChjang** [comment]
This should be fixed by #35410.
